src/globalComponents/userAPIIntegration.js

```diff
@@ -1,24 +1,51 @@
 //@flow strict
 
 import Cookies from 'universal-cookie';
+import getLoginToken from '../globalComponents/getLoginToken.js';
 
 /*
 	This function has no input
 	This function returns a json object of the user
 */
-function getUser() : Promise<Response> {
+function getUser(): Promise<Response> {
 	const cookies = new Cookies();
-	const token = cookies.get('token');
 	const responsePromise: Promise<Response> = fetch('/api/user/getUser/', {
 		method: 'GET',
 		headers: {
 			'Access-Control-Allow-Origin': '*',
 			'Content-Type': 'application/json',
 			'Access-Control-Allow-Credentials': 'true',
-			'x-auth-token': token
+			'x-auth-token': cookies.get('token'),
```

Comment:
please use the getLoginToken() helper method

Reply:
I'll just replace this method.

Reply:
Ok replacing this method takes time. I'll make an issue for it.